lte/gateway/deploy/roles/trfserver/files/magma_interfaces

```diff
@@ -0,0 +1,34 @@
+# interfaces(5) file used by ifup(8) and ifdown(8)
```

Comment:
maybe add a license header here

Reply:
This file is copied into the `/etc/network/interfaces` on the traffic server (as a identical copy of an Ansible step in the `dev_common` role). Does this still need a header? And possibly in the original, as well?